docs/source/common/lightning_module.rst

```diff
@@ -983,6 +983,20 @@ The current logger being used (tensorboard or other supported logger)
         # the particular logger
         tensorboard_logger = self.logger.experiment
 
+loggers
+~~~~~~
+
+The list of loggers currently being used (tensorboard or other supported loggers)
```

Comment:
(tensorboard or other supported loggers) doesn't make sense to me. as a new user i dont know what to do with this info